### PR TITLE
Debug output config variables

### PR DESCRIPTION
**Pull request by TheRealityKing**
Added a debug logger output that prints the contents of the config struct. I was unsure of the desired output format so I went with newline separated.

### CODE REVIEW

File: store/config.go [hunk 1]
```diff
@@ -64,6 +64,41 @@ func (c Config) CreateProductionLogger() *zap.Logger {
 	return logger.CreateProductionLogger(c.RootDir, c.LogLevel.Level)
 }
 
+func (c Config) String() string {
+	fmtConfig := "LOG_LEVEL: %v\n" +
+		"ROOT: %s\n" +
+		"PORT: %s\n" +
+		"USERNAME: %s\n" +
+		"PASSWORD: %s\n" +
```
Comment (dimroc) [inline store/config.go]: As a result of this ending up in output or logs, let's not print or record sensitive data like passwords. So let's just remove this line.

File: cmd/client.go [hunk 2]
```diff
@@ -72,6 +74,12 @@ func logNodeBalance(store *strpkg.Store) {
 	logger.Infow(balance)
 }
 
+func logConfigVariables(config strpkg.Config, debug bool) {
+	if debug {
+		logger.Debug("Environment variables\n", config)
```
Comment (dimroc) [inline cmd/client.go]: `logger.Debug` will already not print if `debug` is false, so you can remove this `if` and the passed `debug bool`.

**Comment by dimroc** [comment]
In order for us to get code coverage on your new code, we'll need to run the test with debug set to true. It's not immediately obvious how to do this, so I've made the changes you need to make in `TestClient_RunNode` below. Obviously open to improvements and suggestions.

```golang
func TestClient_RunNode(t *testing.T) {
	app, _ := cltest.NewApplicationWithKeyStore() // cleanup invoked in client.RunNode
	eth := app.MockEthClient()
	eth.Register("eth_getTransactionCount", `0x1`) // Needed for application to start

	r := &cltest.RendererMock{}
	var called bool
	auth := cltest.CallbackAuthenticator{Callback: func(*store.Store, string) { called = true }}
	client := cmd.Client{
		Renderer:   r,
		Config:     app.Store.Config,
		AppFactory: cltest.InstanceAppFactory{App: app},
		Auth:       auth,
		Runner:     cltest.EmptyRunner{}}

	set := flag.NewFlagSet("test", 0)
	set.Bool("debug", true, "") // set debug to true
	c := cli.NewContext(nil, set, nil)

	assert.Nil(t, client.RunNode(c)) // correctly assert that RunNode is without error
	assert.True(t, called)
}
```

I've placed comments explaining the need for each change. No need to include them in your PR.

**Comment by TheRealityKing** [comment]
Added the changes that you recommended. I would not have been able to figure this out without your help!

**Comment by dimroc** [comment]
Thanks for the PR @TheRealityKing!

**Pull request merged**